From 7a794ce0d1e28f191e113742ce1604334b4187f2 Mon Sep 17 00:00:00 2001
From: Jordi Mallach <jordi@debian.org>
Date: Fri, 28 Jun 2013 15:01:40 +0200
Subject: [PATCH] Add -lm to test programs to fix build failure.

Add -lm to test programs that use sin(), to fix a build failure with recent
binutils.

Signed-off-by: Jordi Mallach <jordi@debian.org>
---
 test/Makefile.am | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/Makefile.am b/test/Makefile.am
index 8705402..c1896f8 100644
--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -7,8 +7,10 @@ check_PROGRAMS=control pcm pcm_min latency seq \
 
 control_LDADD=../src/libasound.la
 pcm_LDADD=../src/libasound.la
+pcm_LDFLAGS= -lm
 pcm_min_LDADD=../src/libasound.la
 latency_LDADD=../src/libasound.la
+latency_LDFLAGS= -lm
 seq_LDADD=../src/libasound.la
 playmidi1_LDADD=../src/libasound.la
 timer_LDADD=../src/libasound.la
-- 
1.8.3.1

